packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -36,6 +36,54 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
     return path != null ? PickedFile(path) : null;
   }
 
+  @override
+  Future<List<PickedFile>?> pickMultiImage({
+    double? maxWidth,
+    double? maxHeight,
+    int? imageQuality,
+  }) async {
+    List<dynamic>? paths = await _pickMultiImagePath(
```

Comment:
nit: could be marked `final`:

```suggestion
    final List<dynamic>? paths = await _pickMultiImagePath(
```